src/Components/Stack/index.tsx

```diff
@@ -1,59 +1,118 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-
-import { getStack } from '@Selectors';
+import { getStack, getStackNewsIdList, getNews } from '@Selectors';
 import { Card } from '@Components/Basic';
-
 import { IStack } from './Stack';
+import { NewsItemList } from './NewsItems';
 
 const Stack: React.FunctionComponent<IStack.IProps> = ({ stackId }) => {
   const stack = useSelector(getStack(stackId));
   if (!stack) return <div />;
 
+  const newsIdList = useSelector(getStackNewsIdList(stackId));
+
   return (
-    <Card>
+    <Card styles={{ paddingTop: '1px', overflow: 'hidden', paddingBottom: '0' }}>
       <div className="stack">
-        {stack.order ? <span className="order light-font">{stack.order}</span> : null}
-        <span>{stack.title}</span>
-      </div>
-      <style jsx>
-        {`
-          .stack {
-            position: relative;
-            padding: 0;
-          }
-
-          .order {
-            font-size: 4.25rem;
-            height: 4.25rem;
-            line-height: 1;
-            position: absolute;
-            right: calc(100% - 1.25rem);
-            top: 0.5rem;
-            font-family: 'Times New Roman', Times, serif;
-            cursor: pointer;
-            font-weight: 900;
-            transition: all 0.2s;
-          }
-
-          @media (max-width: 600px) {
-            .stack {
-              padding: 0.75rem 1rem;
+        <div className="stack-main">
+          <div className="title-area">
+            {stack.order ? (
+              <span className="order-redesigned">
+                <p>{stack.order}</p>
+              </span>
+            ) : null}
+            <h2>{stack.title}</h2>
+          </div>
+
+          <div className="content-area">
+            <p>{stack.description}</p>
+          </div>
+        </div>
+
+        <hr />
+
+        <NewsItemList newsIdList={newsIdList} />
+
+        <style jsx>
+          {`
+            @import url('https://fonts.googleapis.com/css?family=Lexend+Giga&display=swap');
```

Comment:
Font importing should in `public/static/css/common.scss`. We should not use Google Fonts.